index.html

```diff
@@ -423,6 +423,24 @@ <h2>
                       abort these steps. 
                     </p>
                   </li>
+                  <li>
+                    <p>If the <a>current settings object</a>'s's [=relevant global object=]'s [=associated Document=]'s
+                      [=top-level browsing context=] is losing <a data-cite="!HTML/#gains-focus">focus</a>,
+                      queue a global task on the <a data-cite="!HTML/#user-interaction-task-source">user interaction task source</a>
```

Comment:
Which source would that be? Note that focus could also be lost by a new application launching and grabbing it. There need not necessarily be a user action that triggered this focus change. (Or am I misreading this paragraph? It's a very difficult paragraph to read. Could you simplify it? Maybe a "let x be..." and then use it?)

Reply:
The task source is the one used to queue mouse/keyboard events. Maybe we should use another task source, but I do not think it matters much in practice.